hardhat.config.ts

```diff
@@ -88,7 +97,11 @@ const config: HardhatUserConfig = {
     currency: 'USD',
   },
   etherscan: {
-    apiKey: etherscanApiKey,
+    apiKey: {
```

Comment:
These were added to allow the @nomiclabs/hardhat-etherscan verification plugin to work on polygon